discord/permissions.py

```diff
@@ -50,7 +50,7 @@ class Permissions:
         permissions via the properties rather than using this raw value.
```

Comment:
Documentation is changed as of https://github.com/Rapptz/discord.py/pull/193/commits/a5c0924301233fdfe2fbb8abb0fe66dd28b98a8b